src/tokens/size/sizing.js

```diff
@@ -29,6 +29,12 @@ module.exports = {
           comment: 'The minimum space to display a default card in a grid.',
         },
       },
+      logo_group: {
+        item: {
```

Comment:
Should this be `item_width` so that it's clear it represents a width without reading the comment?